lib/gitlab/ci/templates/pipeline/AnsiblePipeline.yml

```diff
@@ -26,48 +11,13 @@ default:
 
 include:
   - template: Security/SAST-IaC.gitlab-ci.yml
+  - remote: https://raw.githubusercontent.com/chesapeaketechnology/gitlab-templates/move-ansible-to-mega-linter/lib/gitlab/ci/templates/jobs/lint/MegaLinter.yml
```

Comment:
Change move-ansible-to-mega-linter to release/1.3 before merging into the release/1.3 branch.